docs/design.md

```diff
@@ -157,9 +158,21 @@ Some known limitations of currently supported platforms:
 
 - No support for preformatted text in cards (blocks) - while text can be rendered as `monospace`, it still does not
   preserve whitespace, so it's not suitable for aligning output into text tables and the like.
-- While text-only messages *do* support Markdown preformatted text, the text is wrapped to a max width of 69
+- While text-only messages _do_ support Markdown preformatted text, the text is wrapped to a max width of 69
   characters regardless of how large the client's window is.
 - No table functionality in cards (blocks). The `ColumnSet` card layout feature allows for wrapping content across
   multiple columns, but does not provide for any alignment across columns, so it's not suitable for tables.
 
+## Settings
+
+The setting of `send_all_messages_private` within the configuration applied within the Nautobot config is used to send all messages as private messages. The messages will be sent in private when this is set to `True`. The default setting is `False`, which is the default behavior for several message settings.
+
+### Settings - Platform Support of Settings
+
+> This table represents the platform support of particular settings
 
+| Platform | send_all_messages_private |
+| Slack | ✅ |
+| MS Teams | ❌ |
+| WebEx | ❌ |
+| Mattermost | ✅ |
```

Comment:
This isn't rendering as a table - I think you need to add a `| -------- | ------------------------- |` after the initial line:

![image](https://user-images.githubusercontent.com/5603551/155735633-a398073a-29f1-4b5c-b59f-b5c861bbfccd.png)